### PR TITLE
combine #260 and #246; remove driver (now in Lotus)

### PR DESCRIPTION
**Pull request by raulk**
Fixes #260.
Fixes #246.

---

Also, remove redundant driver from here.

### CODE REVIEW

File: tvx/examine.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ import (
 	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
```
Comment (willscott) [inline tvx/examine.go:14]: should we leave it as a 'todo' to migrate these specs-actors' references to the stable versions?

**Review by willscott — APPROVED** [review]

File: tvx/examine.go (hunk 1 above)
Comment (raulk) [reply tvx/examine.go:14]: Oh good point, this needs to be migrated!

**Review by nonsense — APPROVED** [review]
overall lgtm

**Comment by raulk** [comment]
Merging this to close it out, but ultimately `tvx` is being from here and ported over to Lotus in https://github.com/filecoin-project/lotus/pull/4064.

**Pull request merged**